lang/extract_json_strings.py

```diff
@@ -21,7 +21,62 @@ def __str__(self):
     "LOADING_ORDER.md"
 }
 
-# don't parse this files. Full related path.
+# these are official mods need to be localized
+# list could be generated by bash command: (in data/mods directory)
+# for i in $(ls -l1); do echo -n '    "'; echo -n $i; echo '",'; done
+# and remove things like replacements.json manually
+official_mods = {
+    "Animatronics",
```

Comment:
3rd-party mods may also have a `maintainers` field. Thus it's difficult to determine whether a mod is official provided without adding an extra mark. Or remind contributors to maintain this list if a new mod is added in or a mod is removed?